samples/xml/dms/src/main/java/sample/dms/AbstractElement.java

```diff
@@ -40,7 +40,7 @@
 	protected AbstractElement() {
 		this.name = "/";
 		this.parent = null;
-		this.id = Long.valueOf(-1);
+		this.id = (long) -1;
```

Comment:
Any reason that you are casting here instead of just doing:

```suggestion
		this.id = -1L;
```

I'm seeing this here and in a few other places in the PR.

Reply:
I've just applied IntelliJ's quick-fix to the whole project.

Reply:
I've corrected the code accordingly and opened https://youtrack.jetbrains.com/issue/IDEA-219831 to (hopefully) prevent this in the future.